core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java

```diff
@@ -228,11 +229,7 @@ public static GeneticProfile loadGeneticProfileFromMeta(File file) throws IOExce
         String geneticAlterationTypeString = properties.getProperty("genetic_alteration_type");
         String datatype = properties.getProperty("datatype");
         String genomeBuild = properties.getProperty("reference_genome_id");
-        int referenceGenomeId = 1;
-        if (genomeBuild != null && !genomeBuild.isEmpty()) {
-            referenceGenomeId = DaoReferenceGenome.getReferenceGenomeIdByName(genomeBuild);
-        }
-
+        int referenceGenomeId = DaoReferenceGenome.getReferenceGenomeIdByName(genomeBuild);
```

Comment:
I meant something else in my last comment. The previous code was almost correct. I mean something like:
```java
int referenceGenomeId = DaoReferenceGenome.getReferenceGenomeIdByName("GRCh37");	
if (genomeBuild != null && !genomeBuild.isEmpty()) {	
           referenceGenomeId = DaoReferenceGenome.getReferenceGenomeIdByName(genomeBuild);	
}
```

Reply:
so, `int referenceGenomeId = DaoReferenceGenome.getReferenceGenomeIdByName("GRCh37");	` instead of `int referenceGenomeId = 1 `

Reply:
Ok, I see latest code has something similar :+1: